src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1580,24 +1578,87 @@ bool Graph::InsertNode(NodePtr parent, NodePtr child, NodePtr node, int parentPo
     return true;
 }
 
+#ifdef CPU_DEBUG_CAPS
```

Comment:
I am wondering if we can do small changes for this debug capability:
1. Is it possbile to extend it for BF16 markup as well?
2. Can we move it into src/utils (into existing or new file)?
3. Can we align env variable naming with other CPU debug variables?
4. It would be nice to add small document descrbing how this itility can be used for debugging (docs/debug_capabilities)

Reply:
OK, changed